--- init.vim	2021-07-28 15:25:56.418701811 -0300
+++ serv_init.vim	2021-07-28 15:25:50.408882062 -0300
@@ -1,15 +1,15 @@
-" function! SendClipBoard()
-"   execute("silent redir! > /tmp/mattf_lcopy_nvim")
-"   execute("silent echo @0")
-"   execute("silent redir END")
-"   execute("silent !lcopy /tmp/mattf_lcopy_nvim")
-" endfunction
-" 
-" augroup custom_clipboard
-"   au!
-"   au TextYankPost * :call SendClipBoard()
-" augroup END
-set clipboard=unnamedplus
+function! SendClipBoard()
+  execute("silent redir! > /tmp/mattf_lcopy_nvim")
+  execute("silent echo @0")
+  execute("silent redir END")
+  execute("silent !lcopy /tmp/mattf_lcopy_nvim")
+endfunction
+
+augroup custom_clipboard
+  au!
+  au TextYankPost * :call SendClipBoard()
+augroup END
+" set clipboard=unnamedplus
 set timeoutlen=500
 set viminfo='1000,f1
 set splitbelow
