--- init.vim	2022-03-02 19:52:34.160767473 -0300
+++ init.server.vim	2022-03-03 03:12:58.383114985 -0300
@@ -1,20 +1,20 @@
-" function! SendClipBoard()
-"   execute("silent redir! > /tmp/mattf_lcopy_nvim")
-"   execute("silent echo @0")
-"   execute("silent redir END")
-"   execute("silent !lcopy /tmp/mattf_lcopy_nvim")
-" endfunction
+function! SendClipBoard()
+  execute("silent redir! > /tmp/mattf_lcopy_nvim")
+  execute("silent echo @0")
+  execute("silent redir END")
+  execute("silent !lcopy /tmp/mattf_lcopy_nvim")
+endfunction
 
-" augroup custom_clipboard
-"   au!
-"   au TextYankPost * :call SendClipBoard()
-" augroup END
+augroup custom_clipboard
+  au!
+  au TextYankPost * :call SendClipBoard()
+augroup END
 
-" function ClipboardSend(txt)
-"   execute("!echo " . a:txt . " | lcopy")
-" endfunction
+function ClipboardSend(txt)
+  execute("!echo " . a:txt . " | lcopy")
+endfunction
 
-set clipboard=unnamedplus
+" set clipboard=unnamedplus
 set timeoutlen=500
 set viminfo='1000,f1
 set splitbelow
